src/shared/components/home/home.tsx

```diff
@@ -170,10 +173,12 @@ export class Home extends Component<any, HomeState> {
           wsClient.communityJoin({ community_id: 0 })
         );
       }
+      const taglines = this.state.siteRes.site_view.taglines;
       this.state = {
         ...this.state,
         trendingCommunities: trendingRes.communities,
         loading: false,
+        tagline: taglines.length == 0 ? "" : taglines[Math.floor(Math.random() * taglines.length)].content,
```

Comment:
Add a `getRandomFromList(` function to utils.tsx .

Then if this is an `Option<Vec<Tagline>>`, you could probably do something like:

`tagline: taglines.map(tls => getRandom(tls).map(tl -> tl.content))` 

Something like that.